content/10.methods.md

```diff
@@ -116,12 +116,27 @@ We were able to define a name and nationality for 708,493 people by using the un
 Our Wikipedia-based process returned a nationality or country of origin, which was more fine-grained than the broader regional patterns that we sought to examine among honorees and authors.
 We initially grouped names by continent, but later decided to model our categorization after the hierarchical nationality taxonomy used by [NamePrism](http://www.name-prism.com/about) [@doi:10.1145/3132847.3133008].
 Consequently, we used the following categories: Hispanic (including Latin America and Iberia), African, Israeli, Muslim, SouthAsian, EastAsian, European (non-British, non-Iberian), and CelticEnglish (including United States, Canada, and Australia).
+Table @tbl:example_names shows the size of the training set for each of these regions, as well as examples of PubMed author names which classified to that region with greater than 95% probability.
 Our dataset, which we term Wiki2019, is available at <https://github.com/greenelab/wiki-nationality-estimate/master/data/annotated_names.tsv>.
+Full information about which countries comprised each region can be found at <https://github.com/greenelab/wiki-nationality-estimate/master/data/country_to_region.tsv>.
+
+| Region | Training Size | Sample Names |
+| ------ | ------------- | ------------ |
+| CelticEnglish | 280644 | Julie S. Miller, Jesse A. Livezey, Jeremy C Simpson, Chris Smith, Thomas M Drudge |
```

Comment:
In the Wiki2019-LSTM figure, we add a space between works in the regions, and we should probably do that here too.